classes/scripts/runner.php

```diff
@@ -244,6 +245,16 @@ public function useConfigFile($path)
 		return $this->includeConfigFile($path, function($path) use ($script, $runner) { include_once($path); });
 	}
 
+	public function useConfigCallable(callable $callback)
```

Comment:
but, there a method called useConfigFile. would'nt it be consistent to name it the same way ?